### DISCUSSION TITLE
How to build from source

### DISCUSSION THREAD

**Discussion by tbayart** [Tyrrrz/QuickJson#3] [Q&A] [answered]
Hi,

I checked out the repository but i have 2 issues.
First one is it fail to build, i get 118 errors from QuickJson.Test.
The first one lines 97 to 98
`            """
            "foo"
            """
`
produce syntax errors.
It looks like it's because the double quotes are not escaped as usual and maybe i have to enable a visual studio option ?

The second one is the QuickJson sources are not visible.
Do i have to change the **visible** flag from **QuickJson.csproj**'s Content item ?

edit: I using Visual Studio Community 2022 v17.1.0

**Comment by Tyrrrz** [comment, marked as answer]
> It looks like it's because the double quotes are not escaped as usual and maybe i have to enable a visual studio option ?

You need a newer Visual Studio. Those features were added in .NET 7.

> The second one is the QuickJson sources are not visible.

It should be visible within the project itself 🤔 
Otherwise, yes, setting `Visible` could help probably.

**Reply by tbayart** [reply]
Hi Tyrrrz,

I upgraded VS to 17.5.3 (latest available) and it fixed the escaping issue with quotes.

For the hidden files, i reminded there is an option in solution explorer : Show All files
Hidden files appeared and everything looks better.

Thanks for your help
